src/ralph/settings/base.py

```diff
@@ -287,9 +287,28 @@ def str_to_bool(s: str) -> bool:
         }
     })
 
+REDIS_MASTER_IP = None
+REDIS_MASTER_PORT = None
+
+REDIS_SENTINEL_ENABLED = os.environ.get('REDIS_SENTINEL_ENABLED', 'false').lower() == 'true'  # noqa: E501
+if REDIS_SENTINEL_ENABLED:
+    from redis.sentinel import Sentinel
+
+    # SENTINEL_HOSTS env variable format: host_1:port;host_2:port
+    SENTINEL_HOSTS = os.environ['SENTINEL_HOSTS'].split(';')
+    REDIS_CLUSTER_NAME = os.environ['REDIS_CLUSTER_NAME']
+
+    sentinel = Sentinel(
+        [tuple(s_host.split(':')) for s_host in SENTINEL_HOSTS],
+        socket_timeout=0.2
+    )
+    REDIS_MASTER_IP, REDIS_MASTER_PORT = sentinel.discover_master(
+        REDIS_CLUSTER_NAME
+    )
+
 REDIS_CONNECTION = {
-    'HOST': os.environ.get('REDIS_HOST', 'localhost'),
-    'PORT': os.environ.get('REDIS_PORT', '6379'),
+    'HOST': REDIS_MASTER_IP if REDIS_MASTER_IP else os.environ.get('REDIS_HOST', 'localhost'),  # noqa: E501
```

Comment:
why not `REDIS_MASTER_IP or os.environ.get('REDIS_HOST', 'localhost')`?

Reply:
you're right, `or` looks better